README.md

```diff
@@ -6,6 +6,43 @@ GitHub Action for `mesheryctl mesh validate` for SMI conformance - https://meshe
 
 <div>&nbsp;</div>
 
+## Example Configuration
+```yaml
+name: Meshery
+on:
+  push:
+    branches:
+      - 'master'
+
+jobs:
+  job1:
+    name: everything
+    runs-on: ubuntu-latest
+    steps:
+      - name: Deploy k8s
+        uses: manusa/actions-setup-minikube@v2.4.1
+        with:
+          minikube version: 'v1.21.0'
+          kubernetes version: 'v1.20.7'
+          driver: docker
+
+      - name: Install OSM
```

Comment:
@kumarabd @navendu-pottekkat we'll want this to change to `mesheryctl mesh deploy osm` soon.